components/Table/index.js

```diff
@@ -2,4 +2,4 @@ import Table from './Table';
 import PlayerMatchesTable from './PlayerMatchesTable.container';
 
 export default Table;
-export { PlayerMatchesTable };
+export { PlayerMatchesTable, Table };
```

Comment:
I don't know if we need to explicitly export table like that as it is the default export. I don't think it will harm anything though, it might be nice even to have both options.